src/openrct2/core/Compression.cpp

```diff
@@ -138,4 +139,137 @@ namespace OpenRCT2::Compression
         inflateEnd(&strm);
         return true;
     }
+
+    /*
+     * Modified copy of ZSTD_COMPRESSBOUND / ZSTD_compressBound() from zstd 1.5.7, with the argument
+     * type changed from size_t (which may be only 32 bits) to uint64_t, and removes the error handling.
+     */
+    static uint64_t zstdCompressBound(uint64_t length)
```

Comment:
zstdCompressBound64?

Reply:
I could name it something like that. The current name isn't going to have collisions with the zstd library, since all its function names start with `ZSTD_`, so changing that would only be a stylistic change. I don't see much reason to make the change, though. If I change the name of this function, we should also rename [`zlibCompressBound`](https://github.com/OpenRCT2/OpenRCT2/blob/b9c3306c3d7c0644e572873fb2330407d2c4a043/src/openrct2/core/Compression.cpp#L33-L36).